docs/administration.md

```diff
@@ -93,6 +94,71 @@ The Pilosa server should support PQL versioning using HTTP headers. On each requ
 
 When upgrading, upgrade clients first, followed by server for all Minor and Patch level changes.
 
+### Resizing the Cluster
+
+If you need to increase (or decrease) the capacity of a Pilosa server, you can add or remove nodes to a running cluster at any time. Note that you can only add or remove one node at a time; if you attempt to add multiple nodes at once, those requests will be enqueued and processed serially. Also note that during any resize process, the cluster goes into state `RESIZING` during which all read/write requests are denied. When the cluster returns to state `NORMAL` then read/write operations can resume. The amount of time that the cluster stays in state `RESIZING` depends on the amount of data that needs to be moved during the resize process.
+
+#### Adding a Node
+
+You can add a new, empty node to an existing cluster by starting `pilosa server` on the new node with the correct configuration options. Specifically, you must specify the [cluster coordinator](../configuration/#cluster-coordinator) to be the same as the coordinator on the existing nodes. You must also specify a valid [gossip seed](../configuration/#gossip-seed). When the new node starts, the coordinator node will receive a `nodeJoin` event indicating that a new node is joining the cluster. At this point, the coordinator will put the cluster into state `RESIZING` and kick off a resize job that instructs all of the nodes in the cluster how to rebalance data to accomodate the additional capacity of the new node. Once the resize job is complete, the coordinator will put the cluster back to state `NORMAL` and ensure that the new node is included in future queries. 
```

Comment:
pluralize seeds: ...must also specify at least one valid [gossip seed](../configuration/#gossip-seeds) (preferably multiple for redundancy).